modules/core/src/main/scala/org/scalasteward/core/nurture/PullRequestRepository.scala

```diff
@@ -55,6 +55,20 @@ final class PullRequestRepository[F[_]](
       }
       .void
 
+  def getObsoleteOpenPullRequests(
+      repo: Repo,
+      crossDependency: CrossDependency,
+      newVersion: String
+  ): F[List[(Uri, Update)]] =
+    kvStore.get(repo).map {
+      _.getOrElse(Map.empty).collect {
+        case (url, data)
+            if UpdateAlg.isForSameArtifacts(data.update, crossDependency) &&
```

Comment:
I think we should also check `data.state === PullRequestState.Open` here.

Reply:
absolutely, will do